app/assets/stylesheets/basic.scss

```diff
@@ -1 +1,27 @@
+#background {
+  background: url("http://www.onefivesports.com/wp-content/uploads/2012/06/basket_ball_court1.png")
+              no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size, min-height